src/packcc.c

```diff
@@ -2321,6 +2321,10 @@ static bool_t parse(context_t *ctx) {
             "}\n"
             "#endif /* defined __GNUC__ && defined _WIN32 */ \n"
             "#endif /* !_MSC_VER */\n"
+            "\n"
+            "#define DBG_EVALUATE 0\n"
+            "#define DBG_MATCH 1\n"
+            "#define DBG_NOMATCH 2\n"
```

Comment:
It's preferable to use the prefix `PCC_` as below:

    #define PCC_DBG_EVALUATE 0
    #define PCC_DBG_MATCH    1
    #define PCC_DBG_NOMATCH  2

`PCC_` works as a name space for PackCC.